Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
The event could be used to remove the field from the mapping based on a configuration node value that would trigger a deprecation notice if not set to true, and would be ignored in next major version.